packages/d3fc-webgl/src/series/glLine.js

```diff
@@ -140,6 +131,39 @@ export default () => {
         return draw;
     };
 
+    draw.defined = (...args) => {
+        const builder = program.buffers().attribute(definedAttrib);
+
+        const definedArray = new Float32Array(args[0].length * 4);
+        const values = args[0];
+
+        for (let i = 0; i < values.length; i += 1) {
+            const val = values[i];
+            const lastVal = i === 0 ? val : values[i - 1];
```

Comment:
I think it would be more consistent with the rest of the code if this was called `prevVal`